src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/jenkins-whitelist

```diff
@@ -4,6 +4,8 @@ method hudson.model.AbstractBuild getEnvironments
 staticMethod hudson.model.Environment create hudson.EnvVars
 method hudson.model.Item getParent
 method hudson.model.ModelObject getDisplayName
+method hudson.model.Run getCause java.lang.Class
+method hudson.model.Run getCauses
```

Comment:
Hardly useful on its own, since you would need to then whitelist fields in the cause.